concourse/scripts/frontend-unit-tests

```diff
@@ -6,5 +6,4 @@ source "$script_dir/util-lib.sh"
 skip_if_no_changed_file_match "^(frontend|concourse)"
 
 ./install_frontend_dependencies.sh root
-./build_full_frontend.sh
 ./run_frontend_unit_test.sh
```

Comment:
is it normal that the build has been deleted here?

Reply:
yes, it doesn't do anything unit tests don't require it

Reply:
@Skylsmoi we don't need to build frontend_lib for frontend app' unit tests anymore?

Reply:
actually it does D:
but only to build frontend_lib as standalone

will fix

Reply:
fixed

Reply:
@raphj yes, but only as standalone (didn't see your comment)